hathor/prometheus.py

```diff
@@ -136,10 +157,17 @@ def set_new_metrics(self) -> None:
         for metric_name in METRIC_INFO.keys():
             self.metric_gauges[metric_name].set(getattr(self.metrics, metric_name))
 
+        self._set_new_tx_storage_metrics()
         self._set_new_peer_connection_metrics()
 
         write_to_textfile(self.filepath, self.registry)
 
+    def _set_new_tx_storage_metrics(self) -> None:
```

Comment:
Maybe rename to `_set_rocksdb_tx_storage_metrics()`?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/485/commits/2b4f53ebae7134bc81952faae6e65d2f14b1b2a6